mu-plugins/blocks/global-header-footer/postcss/header/menu.pcss

```diff
@@ -315,8 +321,10 @@ Source: https://github.com/WordPress/gutenberg/blob/3b2eccc289cfc90bd99252b12fc4
 		}
 
 		/* Submenu indentation when there's no background. */
-		left: -1px; /* Border width. */
-		top: 100%;
+
+		/* Changed from source — left has no border, top needs 4px for current item border. */
+		left: 0;
+		top: calc(100% + 4px);
```

Comment:
Should the border-width be parameterized?

Reply:
Oh, It already is, I can use `--active-menu-item-border-height` - good catch.